src/wallet/wallet2.cpp

```diff
@@ -777,12 +824,28 @@ static uint64_t decodeRct(const rct::rctSig & rv, const crypto::key_derivation &
   }
 }
 //----------------------------------------------------------------------------------------------------
+bool wallet2::wallet_generate_key_image_helper_export(const cryptonote::account_keys& ack, size_t multisig_key_index, const crypto::public_key& out_key, crypto::key_image& ki) const
```

Comment:
It's using ack AFAICS.

Reply:
I mean, currently this function is literally equivalent to`cryptonote::generate_multisig_key_image`. So my point is, why not either 1) drop it and directly use `cryptonote::generate_multisig_key_image`, or 2) drop `ack` from the argument list and call `cryptonote::generate_multisig_key_image(get_account().get_keys(), multisig_key_index, out_key, ki)`.

IMO the former makes more sense, since the error checking is done at the caller's side (`export_multisig`).